doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -774,10 +774,10 @@ Strategies for rewriting
 Usage
 ~~~~~
 
-.. tacn:: rewrite_strat @rewstrategy {? in @ident }
+.. tacn:: rewrite_strat {| @rewstrategy | {+; @rewstrategy } } {? in @ident }
```

Comment:
```suggestion
.. tacn:: rewrite_strat {+; @rewstrategy } {? in @ident }
```
right?

Reply:
It is produced automatically by docgram. It seems to correspond to a bug fix made by @jfehrle, but I'm not sure.

Reply:
I'll take a look

Reply:
When I download the branch, I see Jason's suggested change in the file.  Hugo, perhaps you did `doc_gram` rather than `doc_gram_rsts` or forget to include the rst file in your commit.

Reply:
I don't know. At least, the branch is right now synchronous.